test/linalg/tridiag.jl

```diff
@@ -119,7 +119,7 @@ for elty in (Float32, Float64, Complex64, Complex128, Int)
             invFsv = Fs\vv
             x = Ts\vv
             @test x ≈ invFsv
-            @test full(full(Tldlt)) ≈ Fs
+            @test Array(Array(Tldlt)) ≈ Fs
```

Comment:
Derp is what's going on here. Fixed on push. (`Array(::LDLt)` returns an `Array`.) Thanks!